### PR TITLE
Public API changes for 1.1.0 release

### PR DESCRIPTION
**Pull request by stealthcode**

### CODE REVIEW

File: src/main/java/rx/subjects/SerializedSubject.java [hunk 1]
```diff
@@ -69,6 +68,7 @@ public void onNext(T t) {
     public boolean hasObservers() {
         return actual.hasObservers();
     }
+<<<<<<< HEAD
```
Comment (akarnokd) [inline src/main/java/rx/subjects/SerializedSubject.java]: Merge conflict not resolved; this causes compilation error.

**Comment by stealthcode** [comment]
This should be merged immediately after the next minor release (1.0.17). The plan is to have a minor release that is identical to the following major release except for changes to the public api (which should not substantially change functionality).

File: src/main/java/rx/subjects/SerializedSubject.java (hunk 1 above)
Comment (stealthcode) [inline src/main/java/rx/subjects/SerializedSubject.java]: My mistake. Thanks for the clue.

**Comment by akarnokd** [comment]
:+1:

**Comment by stevegury** [comment]
:+1:

**Comment by stealthcode** [comment]
Now we need to update the CHANGES.md to reflect the public api changes... That might have been easier in the old format of 1 PR per change. I'll probably do sub-bullets under the PR.

**Comment by mattbishop** [comment]
So Release Notes for 1.1.0 state that Single has been promoted out of Beta, but it looks like it is not actually out of beta.

**Pull request merged**